client/crowi.ts

```diff
@@ -604,9 +604,9 @@ $(function() {
     }
 
     // header
-    var $headerWrap = $('#page-header').parent()
+    var $headerWrap = $('.header-wrap')
     if ($headerWrap.length > 0) {
-      const headerHeight = $('.crowi-header').outerHeight() || 0
+      const headerHeight = $('.v2-crowi-header').outerHeight() || 0
```

Comment:
👍 Nice fix.
But I found the sticky header broken in case the page is not public. Can you also fix it?

![image](https://user-images.githubusercontent.com/29064/66360629-c92d0000-e9b6-11e9-9054-2fb294cda3a4.png)

Reply:
Oh, I just fixed it.